Gemfile.lock

```diff
@@ -44,9 +46,9 @@ GEM
     celluloid-io (0.16.2)
       celluloid (>= 0.16.0)
       nio4r (>= 1.1.0)
-    chef (12.8.1)
+    chef (12.9.41)
```

Comment:
just a note that chef 12.9 seemed unstable to me (node attribute.each on an array fails), so we might be best pinning to ~> 12.8.0